sphinx_gallery/gen_gallery.py

```diff
@@ -180,11 +182,22 @@ def generate_gallery_rst(app):
 
         computation_times += this_computation_times
 
+        key = None
+        if gallery_conf['examples_dirs_order'] is not None:
+            base_examples_dir = os.path.basename(examples_dir)
```

Comment:
cool - I think this is a good point, let me see if I can work something out per your suggestions that's more stable